### PR TITLE
Add KMyMoney

### PR DESCRIPTION
**Pull request by Mailaender**
### Please confirm your submission meets all the criteria

- [X] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [X] My pull request follows the instructions at [App Submission][submission].
- [X] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [X] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [X] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:** https://bugs.kde.org/show_bug.cgi?id=445050
- [ ] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [X] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[maint]: https://docs.flathub.org/docs/for-app-authors/maintanance
[submission]: https://docs.flathub.org/docs/for-app-authors/submission

### CODE REVIEW

**Comment by Mailaender** [comment]
bot, build org.kde.kmymoney

**Comment by hfiguiere** [comment]
cc @flathub/kde

File: org.kde.kmymoney.json [hunk 1]
```diff
@@ -0,0 +1,314 @@
...
+        "/share/man"
+    ],
+    "cleanup-commands": [
+        "/app/cleanup-BaseApp.sh"
+    ],
+    "finish-args": [
+        "--share=ipc",
+        "--share=network",
+        "--socket=x11",
+        "--socket=wayland",
```
Comment (hfiguiere) [inline org.kde.kmymoney.json:26]: fallback-x11?

Comment (Justinzobel) [reply org.kde.kmymoney.json:26]: Yeah fallback-x11.

**Comment by Justinzobel** [comment]
```
OUTDATED: kmymoney-5.1.3.tar.xz
 Has a new version:
  URL:       https://download.kde.org/stable/kmymoney/5.1.1/src/kmymoney-5.1.1.tar.xz
  ```
  Is that right :rofl:

**Comment by Justinzobel** [comment]
Hopefully I've fixed up https://release-monitoring.org/project/8917/ as it now shows 5.1.3 as the latest.

**Comment by Mailaender** [comment]
bot, build org.kde.kmymoney

**Comment by Mailaender** [comment]
bot, build org.kde.kmymoney

**Comment by Mailaender** [comment]
bot, build org.kde.kmymoney

**Comment by hfiguiere** [comment]
you need to copy the config.guess and config.sub to opensp for building

**Comment by tsdgeos** [comment]
kmymoney is quite a specialized app, I would suggest you contact the authors so they can give it a bit of testing before we merge this

**Comment by ostroffjh** [comment]
In response to tsdgeos:  after some file access issues I believe are related to my base flatpak install, running under both Xorg and Wayland is good.  Testing wasn't very deep, but covered most common activities and screens.  Note I'm not a KMM developer, but do documentation and user support, but unless one of the developers has something to add, I think this is OK to release.

**Comment by tsdgeos** [comment]
If Jack is happy, i'm happy

**Comment by wrobelda** [comment]
Why not use cmake for building libofx?

**Comment by travier** [comment]
> Why not use cmake for building libofx?

https://github.com/flathub/org.kde.kmymoney/pull/1

**Pull request closed without merging**